plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMapping.java

```diff
@@ -35,15 +39,27 @@
 @Test(singleThreaded = true)
 public class TestBigQueryCaseInsensitiveMapping
         // TODO extends BaseCaseInsensitiveMappingTest - https://github.com/trinodb/trino/issues/7864
-        extends AbstractTestQueryFramework
+        extends BaseCaseInsensitiveMappingTest
 {
     private BigQuerySqlExecutor bigQuerySqlExecutor;
+    private Path Mappingfile;
+
+    @Override
+    protected SqlExecutor onRemoteDatabase() {
+        return requireNonNull(bigQuerySqlExecutor,"bigQuerySqlExecutor is null");
```

Comment:
Remove the `requireNonNull`. It cannot be `null` because if the initialisation fails in the `createQueryRunner` method the test setup fails and we'll never reach here.

Same in other files.